packages/tools/src/stateManagement/segmentation/polySeg/Surface/convertContourToSurface.ts

```diff
@@ -0,0 +1,48 @@
+import { Types, eventTarget, triggerEvent } from '@cornerstonejs/core';
+import { getWebWorkerManager } from '@cornerstonejs/core';
+import { ContourSegmentationData } from '../../../../types';
+import { getAnnotation } from '../../../annotation/annotationState';
+import { Events } from '../../../../enums';
+
+const workerManager = getWebWorkerManager();
+
+export async function convertContourToSurface(
```

Comment:
docs please - useful to know that this calls the wasm conversion as a webworker manager and returns the promise when it is done converting, with the provided data.